ExampleApp/Assets/OptimoveSdk/Editor/SetUpXcodeProject.cs

```diff
@@ -107,36 +113,36 @@ private static void AddOptimoveConfig(PBXProject project, string pathToBuiltProj
 
         // Copy the entitlement file to the xcode project
         var entitlementFileName = Path.GetFileName(entitlementsPath);
-        var relativeDestination = targetName + "/" + entitlementFileName;
+        var relativeDestination = Path.Combine(targetName, entitlementFileName);
 
         // Add the pbx configs to include the entitlements files on the project
-        project.AddFile(relativeDestination, entitlementFileName);
+        project.AddFile(relativeDestination, isExtension ? relativeDestination : entitlementFileName);
         project.SetBuildProperty(targetGuid, "CODE_SIGN_ENTITLEMENTS", relativeDestination);
 
         return relativeDestination;
     }
 
-    private static void SetModuleMap(PBXProject project, string unityTarget, string buildPath)
+    private static void SetModuleMap(PBXProject project, string unityTarget, string pathToBuiltProject)
     {
         // Modulemap
         project.AddBuildProperty(unityTarget, "DEFINES_MODULE", "YES");
 
-        var moduleFile = buildPath + "/UnityFramework/UnityFramework.modulemap";
+        var moduleFile = Path.Combine(pathToBuiltProject, "UnityFramework", "UnityFramework.modulemap");
         if (!File.Exists(moduleFile))
         {
-            FileUtil.CopyFileOrDirectory("Assets/Plugins/iOS/UnityFramework.modulemap", moduleFile);
-            project.AddFile(moduleFile, "UnityFramework/UnityFramework.modulemap");
+            FileUtil.CopyFileOrDirectory("Assets/Plugins/iOS/UnityFramework.modulemap", moduleFile.Replace("\\", "/"));
```

Comment:
Why are paths here like this and not using Path.Combine?

Reply:
https://docs.unity3d.com/ScriptReference/FileUtil.CopyFileOrDirectory.html -- `All file separators should be forward ones "/".`

In general this exists to build on Windows, but 
1. it was, obviously, untested, so, it's likely it would fail if someone tries
2. the hope is that if you build your xcode project you are on ios
3. theoretically it should be mostly correct